gpu-operator-certified/instance/INFO.md

```diff
@@ -0,0 +1,48 @@
+# GPU Notes
```

Comment:
Any reason for info.md instead of a readme?

Also, it would be great to have some details about the instance overlays and the purpose of each one.  Doesn't need to get into too much detail but a little info on these would be awesome.  I would love to be able to point people here and it to have enough context to help them decide which one they should be using.

Reply:
Yeah, I will update this with a README.

Reply:
@strangiato Let me know if the update is what you were looking for.